gbn/docker-compose.yml

```
@@ -85,12 +87,15 @@ services:
       - "traefik.enable=true"
       - "traefik.http.routers.gbn_connector.rule=PathPrefix(`/gbn-connector`)"
       - "traefik.http.services.gbn_connector.loadbalancer.server.port=8080"
+      - "traefik.http.routers.gbn_connector.tls=true"
     depends_on:
       - "gbn-connector-db"
     restart: "always"
 
   gbn-connector-db:
+    container_name: bridgehead-gbn-connector-db
     image: "postgres:10.17"
```

```suggestion
    image: "postgres:10"
```

see https://www.postgresql.org/support/versioning/